demisto_sdk/commands/run_cmd/runner.py

```diff
@@ -48,12 +55,29 @@ def run(self):
             log_ids = None
             print_error(str(err))
 
-        if self.debug:
+        if self.debug and not self.json2outputs:
             if not log_ids:
                 print_warning('Entry with debug log not found')
             else:
                 self._export_debug_log(log_ids)
 
+        if self.json2outputs:
+            if not log_ids:
+                print_warning('Entry with debug log not found')
+            else:
+                raw_output_json = self._return_raw_outputs_from_log(log_ids)
+                with tempfile.NamedTemporaryFile(mode='w+') as f:
+                    if isinstance(raw_output_json, dict):
+                        f.write(json.dumps(raw_output_json))
+                    if isinstance(raw_output_json, list):
+                        f.write(json.dumps(raw_output_json[0]))
+                    f.seek(0)
+                    file_path = f.name
+                    command = self.query.split(' ')[0]
+                    print("What would you like the base path for the outputs that the script generates to be?")
+                    prefix = input()
```

Comment:
we usually shy away from giving inputs in the middle of a demisto-sdk command run, this way we can use them as part of a different automation, just as you did with `json_to_outputs`.
put it as an arg, and think of a default path.